Dockerfile.test-infra

```diff
@@ -22,5 +22,7 @@ RUN pip3 install --no-cache-dir -r /tmp/requirements.txt && \
 # TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
 RUN chmod uga+w /etc/passwd
 
+RUN curl -Lo oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/${OPENSHIFT_VERSION:-4.5}/linux/oc.tar.gz && tar -C /usr/local/bin -xf oc.tar.gz && rm -f oc.tar.gz
```

Comment:
no need right?

Reply:
still needed :(

Reply:
The image should be passed by an environment variable like in dev-scripts to support Prow different images
Look in here:
https://github.com/openshift-metal3/dev-scripts/blob/master/03_build_installer.sh#L11
https://github.com/openshift-metal3/dev-scripts/blob/master/ocp_install_env.sh#L24

Reply:
openshift version is not enough? I think it is enough